WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModel.swift

```diff
@@ -44,6 +44,14 @@ final class PaymentMethodsViewModel: ObservableObject {
         paymentLink != nil
     }
 
+    var showUpsellCardReaderFeatureBanner: Bool {
+        return cardPresentPaymentsConfiguration.isSupportedCountry
+    }
+
+    var purchaseCardReaderUrl: URL {
+        cardPresentPaymentsConfiguration.purchaseCardReaderUrl()
+    }
```

Comment:
Perhaps we should add inline comments here? What do you think of something like:

```
/// Defines if the Card Reader upsell banner should be shown based on country eligibility
///
var showUpsellCardReaderFeatureBanner: Bool {
	return cardPresentPaymentsConfiguration.isSupportedCountry
}

/// Returns the URL where the merchant can purchase a card reader based on store country code
///
var purchaseCardReaderUrl: URL {
	cardPresentPaymentsConfiguration.purchaseCardReaderUrl()
}
```